src/explorer/SubscriptionTreeItem.ts

```diff
@@ -112,6 +116,30 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         const createdNewAppMsg: string = `Created new web app "${siteClient.fullName}": https://${siteClient.defaultHostName}`;
         ext.outputChannel.appendLine(createdNewAppMsg);
 
+        if (wizardContext.appInsightsComponent && wizardContext.appInsightsComponent.instrumentationKey) {
```

Comment:
Wait turns out I only reviewed one commit not the whole PR 😂

This should be a part of SiteCreateStep

https://github.com/microsoft/vscode-azuretools/blob/master/appservice/src/createAppService/SiteCreateStep.ts#L63

Rather than making a separate call to update app settings after the web app is created